kubejs/server_scripts/mods/modern_industrialization/boss_crusher.js

```diff
@@ -0,0 +1,48 @@
+// -----------------------------------------
+// CREATED BY STATIC FOR USE IN
+// STATECH INDUSTRY
+// -----------------------------------------
+
+ServerEvents.recipes(e => {
+    // -- MOD NAMESPACE UTILITY FUNCTIONS -- // 
+    let st = (id) => `statech:modern_industrialization/boss_crusher/${id}`;
+    let mi = (id) => `modern_industrialization:${id}`;
+    let mc = (id) => `minecraft:${id}`;
+    let cr = (id) => `create:${id}`;
+    let kj = (id) => `kubejs:${id}`;
+
+    // -- CUSTOM RECIPE UTILITY FUNCTION -- //
+    let boss_crusher = (id, eu, duration, item_inputs, item_outputs, fluid_inputs, fluid_outputs) => {
+        let newRecipe = {
+            type: mi('boss_crusher'),
+            eu: eu,
+            duration: duration
+        }
+
+        if (item_inputs)
+            newRecipe['item_inputs'] = item_inputs;
+        if (item_outputs)
+            newRecipe['item_outputs'] = item_outputs;
+        if (fluid_inputs)
+            newRecipe['fluid_inputs'] = fluid_inputs;
+        if (fluid_outputs)
+            newRecipe['fluid_outputs'] = fluid_outputs;
+
+        e.custom(newRecipe).id(id);
+    }
+
+
+    // -- WITHER MODEL -- //
+    boss_crusher(
+        st('wither_model'),
+        8,
+        600,
+        [ { amount: 1, item: kj('wither_model') } ],
+        [
+            { amount: 1, item: mc('ender_pearl'), probability: 0.15 },
+            { amount: 4, item: cr('experience_nugget'), probability: 0.5 }
```

Comment:
Kind of minor, but 4 experience nuggets is a bit low for killing the Wither? Could give the player more at this point.

Reply:
I'll quadruple the experience nuggets.

Reply:
And make it drop them every recipe, since the model is consumed every time.